cli/src/commands/add.ts

```
@@ -0,0 +1,424 @@
+      .getVariableDeclaration("config")
+      ?.getInitializer()
+      ?.getText();
+    
+    const defaultConfig = defaultConfigFile
+      .getVariableDeclaration("config")
+      ?.getInitializer()
+      ?.getText();
+
+    if (existingConfig && defaultConfig) {
+      // Parse configs to objects
+      const existing = eval(`(${existingConfig})`);
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)